lib/color.sh

```diff
@@ -32,6 +32,11 @@ function byellow(){
     echo -e "\033[33m\033[01m$1\033[0m"
 }
 
+## info in green to blue to echo
+function bgreen(){
+    echo -e "\033[34m\033[01m$1\033[0m"
```

Comment:
`\033[34m` is set to color blue, why you name it `bgreen` ?

Reply:
automatic error I use the nightly github copilot and I have no knowledge of the colors code in bash.

Reply:
`34m` is blue.